popSimManu.tex

```diff
@@ -517,13 +525,16 @@ \subsubsection*{ Single-population demographic models.}
  on simulated genomes sampled from a single population,
 under several of the demographic models described above.
 However, these experiments raise the question of what to use as the ``true" population sizes in the case of multi-population models with migration.
-In particular, when simple single-population models are fit to data simulated under such complex models, 
-they are not expected to recover the actual simulated population sizes owing to model misspecification.
+In particular, a simple single-population model that is fit to data simulated
+under a multi-population model, 
+is not expected to recover the actual simulated population sizes,
+because of to model misspecification.
 Instead, we argue that the best one may expect in such a scenario is to infer a model that accurately reflects the
-coalescence time distribution of the simulated model.  In turn, the inverses of these coalescence times (expressed in appropriate units) 
-correspond to population sizes.
+coalescence time distribution of the simulated model.
 Under a multi-population model, the coalescence time distribution is influenced by migration between the target population and populations not analyzed in inference,
 as well as by the ancestral effective population sizes.
+Inverse coalescence rate is then the natural comparison for population size,
+since these are equal in a single-population model.
```

Comment:
this statement is unclear , in particular the use of "natural comparison". Suggestion:
" The inverse coalescence rate thus represents the ``inferable" version of the effective population size in this setting. "

Reply:
Gee, I agree that it's not very precise, but it does try to explain what it means? What if it says "is *a* natural comparison"?  I'm not enthusiastic about your suggestion because it is also vague: *why* is it the inferable version? I was trying to explain (briefly) why this is; I think the easy answer is because they're equal for a single-pop model. The fact that any coal time distribution can be matched by a pop of changing size gives more context, but that was maybe too much detail.

Reply:
I agree that we should keep short and simple, even if it's not 100% complete. My main problem with the current wording is with the use of 'comparison', which seems a bit odd. How about 'analog', or 'parallel'?

Reply:
How about "It is then natural to compare inferred effective population sizes to the inverse coalescence rates of the model used for simualtion, since these are equal in a single-population model." ?  (except: got a better substitute for "natural"?)

Reply:
Maybe "The inverse coalescence rate is commonly _interpreted_ as the effective population size, since these are equal in a single-population model with random mating." ?

Reply:
Both seem fine to me.

Reply:
Sounds good; I'll change that then merge this.